corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js

```diff
@@ -671,7 +671,7 @@ cloudCare.AppView = Backbone.View.extend({
             });
         };
         data.onerror = function (resp) {
-            showError(resp.human_readable_message, $("#cloudcare-notifications"));
+            showError(resp.human_readable_message || resp.message, $("#cloudcare-notifications"));
```

Comment:
This is not really a pre-req, it just makes debugging much easier when there is no `human_readable_message`.